src/client/components/VoyageTeaser.js

```diff
@@ -9,16 +41,33 @@ function VoyageTeaser({
   departure_time,
   target_arrival_time
 }) {
+  const classes = voyage_teaser();
+
   return (
-    <div>
-      <h1>{vesselName}</h1>
-      <label>Departure</label>
-      <h3>{departureFrom}</h3>
-      <p>{departure_time}</p>
-      <label>ETA</label>
-      <h3>{arrivesAt}</h3>
-      <p>{target_arrival_time}</p>
-    </div>
+    <Card className={classes.card}>
+      <CardContent>
+        <Typography className={classes.headerTitle}>{vesselName}</Typography>
+        <CardContent className={classes.summery}>
+          <Typography className={classes.infoTitle}>DEPARTURE</Typography>
+          <Typography className={classes.describeText}>
+            {departureFrom}
+          </Typography>
+          <Typography className={classes.describeText}>
+            {departure_time}
+          </Typography>
+        </CardContent>
+        <CardContent className={classes.summery}>
+          <Typography className={classes.infoTitle}>ETA</Typography>
+          <Typography className={classes.describeText}>{arrivesAt}</Typography>
+          <Typography className={classes.describeText}>
+            {target_arrival_time}
+          </Typography>
+        </CardContent>
+        <CardActions>
+          <Button size="small">Show Details</Button>
```

Comment:
Could you make the following changes to the show details button:

- [ ] Make the font size 1.25rem
- [ ] Align the button to the right
- [ ] Make the font bold

Reply:
In my opinion, 1.25rem for the button is too much but you are the one who decides.
an I didn't understand what you mean by "Use Component", Do you want me to make a new Component for button and import it to the "VoyageTeaser"?